target/linux/ath79/generic/base-files/etc/board.d/02_network

```diff
@@ -190,7 +190,8 @@ ath79_setup_interfaces()
 		ucidef_add_switch "switch0" \
 			"0@eth0" "1:wan" "2:lan"
 		;;
-	comfast,cf-e560ac)
+	comfast,cf-e560ac|\
+	tplink,tl-wr841hp-v3)
 		ucidef_set_interface_wan "eth1"
 		ucidef_add_switch "switch0" \
 			"0@eth0" "1:lan" "2:lan" "3:lan" "4:lan"
```

Comment:
Normal wr841 has inverted port labels:
`"0@eth0" "1:lan:4" "2:lan:3" "3:lan:2" "4:lan:1"`

Are yours correct?

Reply:
Yep, i'm sure it's correct. Plugin cable to ethernet port 1 on router, switch status on luci gui show Lan 1.

Reply:
Perfect, thanks.